### ISSUE TITLE
A Question about Inference Speed

### ISSUE THREAD

**Issue by yanshengjia** [pytorch/fairseq#380]
Recently, I trained two fairseq models with different cnn kernel size, one is 256 kernels and another is 512.

The training script is like
`PYTHONPATH=$FAIRSEQPY:$PYTHONPATH CUDA_VISIBLE_DEVICES="0" python3.6 $FAIRSEQPY/train.py --arch fconv --save-dir $OUT_DIR --encoder-embed-dim 500 --encoder-embed-path $EMBED_PATH --decoder-embed-dim 500 --decoder-embed-path $EMBED_PATH --decoder-out-embed-dim 500 --dropout 0.2 --clip-norm 0.1 --lr 0.25 --min-lr 1e-4 --encoder-layers '[(256,3)] * 7' --decoder-layers '[(256,3)] * 7' --momentum 0.99 --max-epoch 100 --batch-size 32 --no-progress-bar --seed $SEED --data $DATA_BIN_DIR`

When inference with these two models, I found that the model with 512 kernels is faster than the 256 one.

Shouldn't the model with 256 kernels is faster because it has less kernels?

**Comment by myleott** [comment]
Please reopen if the above suggestion didn't help, thanks!
